controllers/deployment_controller.go

```diff
@@ -60,7 +59,7 @@ const (
 
 // Reconcile reconciles a Deployment object
 func (r *DeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log := r.Log.WithValues("deployment", req.NamespacedName)
+	log := r.Log.WithField("deployment", req.NamespacedName)
 
 	deployment := new(csibaremetalv1.Deployment)
 	err := r.Client.Get(ctx, client.ObjectKey{Name: req.Name, Namespace: req.Namespace}, deployment)
```

Comment:
In this file we have comments to increase log level from info to warn after logger changing. Could you please grep repo for other mentions of this issue?

Reply:
Done